Sources/UIComponent/Components/StackComponent.swift

```diff
@@ -127,6 +135,9 @@ extension StackComponent {
         let childRenderer = child.layout(childConstraint)
         mainFreezed += main(childRenderer.size)
         renderers.append(childRenderer)
+        if bound {
+            childConstraint.maxSize = size(main: main(constraint.maxSize) - mainFreezed, cross: cross(childConstraint.maxSize))
+        }
```

Comment:
This will break many of the existing app using UIComponent. If you want to suggest the remaining space to the child, we shouldn't do it this way. Like I said, `HStack` and `VStack` is currently optimized for long feed, and if we force a `maxSize` to be the remaining height, many component will break and be clamped to the `maxSize` here.

I would go against this change because if you want a component to fill you remaining height, you can use the `.flex()` parameter. If you want to read the parent constraint, `UIComponent` does have a `ConstraintReader` for you to be able to read the parent constraint at layout time.